yarn.lock

```diff
@@ -5815,6 +5815,21 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask/multichain-network-controller@npm:^0.4.0":
```

Comment:
Not sure why we have 2 duplicated entries for that package, maybe we should `dedupe`?

Reply:
I only see one

Reply:
There are two copies of `multichain-network-controller`, 0.3.0 and 0.4.0, is that what you mean @ccharly? I agree that we could possibly de-dupe here.

Reply:
Ah ok, I see it now. Was juts looking at the diff. I tried to dedupe but it did not work

Reply:
I can probably force it with a resolution?

Reply:
Probably, but maybe it's okay as is. If we upgrade `bridge-controller` to use `multichain-network-controller` 0.4.0 later, that would also fix this.